src/plugins/ruby.ts

```diff
@@ -10,7 +10,8 @@ function locateRuby(value: string, fromIndex: number) {
 
 const tokenizer: Tokenizer = function (eat, value, silent) {
   const now = eat.now();
-  const match = /^{(.+?)\|(.+?)}/.exec(value);
+  ///const match = /^{(.+?)\|(.+?)}/.exec(value);
+  const match = /^{(.+?)(?<=[^\\|])\|(.+?)}/.exec(value);
```

Comment:
nits: git logで見られるので特に残さなくても大丈夫かと思います

```suggestion
  const match = /^{(.+?)(?<=[^\\|])\|(.+?)}/.exec(value);
```